--- unh-iscsi-2.0/common/user_space.h.orig	2008-05-28 20:54:49.000000000 +0200
+++ unh-iscsi-2.0/common/user_space.h	2024-03-17 14:30:08.472082138 +0100
@@ -48,6 +48,7 @@
 #include <fcntl.h>
 #include <signal.h>
 #include <pthread.h>
+#include <limits.h>
 #include <sys/types.h>
 #include <sys/socket.h>
 #include <sys/times.h>
--- unh-iscsi-2.0/initiator/initiator_utilities.c.orig	2008-05-28 20:54:49.000000000 +0200
+++ unh-iscsi-2.0/initiator/initiator_utilities.c	2024-03-17 14:42:53.027940184 +0100
@@ -224,7 +224,7 @@ static inline void __attribute__ ((no_in
 	this_command->tx_iovlen = 1;
 }
 
-inline int __attribute__ ((no_instrument_function))
+int __attribute__ ((no_instrument_function))
     get_immed_length(struct session *current_session,
 		 struct connection *current_connection,
 		 struct scsi_cmnd *Cmnd)
